lib/types.ts

```diff
@@ -51,7 +58,8 @@ export interface Package {
 
 export interface PluginResponse {
   plugin: PluginMetadata;
-  package: Package;
+  package: Package; // under deprecation, Package is one type of ecosystem hint
+  packages: EcosystemHints[]; // to replace package
   manifestFiles: ManifestFile[];
```

Comment:
I know it's not related to this change, but we might want to remove this field going forward (or now), as the new EcosystemHints[] will provide these.
@mladkau ?

Reply:
I think we would remove it when we restructure how ```packages``` look like exactly, but let's keep the discussion and changes focused please 🙏